src/main/java/com/sublinks/sublinksapi/slurfilter/services/SlurFilterService.java

```diff
@@ -22,7 +22,7 @@ public class SlurFilterService {
   @Transactional
   public void updateOrCreateLemmySlur(String regex) {
 
-    SlurFilter slurFilter = slurFilterRepository.findAll().stream().findFirst().orElse(null);
+    SlurFilter slurFilter = slurFilterRepository.findById(1L).orElse(null);
```

Comment:
What if ID 1 gets deleted? Why not keep it the way it was before and just grab the first record?

Reply:
As otherwise side effects could ruin the platform.